app.py

```diff
@@ -59,14 +84,20 @@ def _required_auth_capability(self):
 
 DEFAULT_HOST = 'search-dss-azul-commons-lx3ltgewjw5wiw2yrxftoqr7jy.us-west-2.es.amazonaws.com'
 DEFAULT_REGION = 'us-west-2'
-DEFAULT_INDEX = 'fb_index'
-DEFAULT_DOCTYPE = 'meta'
 DEFAULT_ACCESS_TOKEN = 'f4ce9d3d23f4ac9dfdc3c825608dc660'
 
-es_index = os.environ.get('ES_INDEX', DEFAULT_INDEX)
+INDEXES = {
+    'data_obj': os.environ.get('DATA_OBJ_INDEX', 'fb_index'),
```

Comment:
This is good and should enable us to configure either working in the same index or a different index easily. Would you please document in the development section how to configure the index or doctype selection?